sim/battle.ts

```diff
@@ -139,7 +138,7 @@ export class Battle {
 
 	trunc: (num: number, bits?: number) => number;
 	clampIntRange: (num: any, min?: number, max?: number) => number;
-
+	toID: (args: any) => ID;
```

Comment:
```suggestion
	toID = toID;
```

This might make it easier than having to restate the definition.